internal/matcher/controller.go

```diff
@@ -45,6 +45,11 @@ func (mc *Controller) Match(ctx context.Context, records []*claircore.IndexRecor
 		return map[string][]*claircore.Vulnerability{}, nil
 	}
 
+	remoteMatcher, matchedVulns, err := mc.queryRemoteMatcher(ctx, interested)
+	if remoteMatcher {
+		return matchedVulns, err
```

Comment:
```suggestion
		if err != nil {
		  log.Error().Err(err).Msg("remote matcher error, returning empty results")
		}
		return map[string][]*claircore.Vulnerability, nil
```

Because http endpoints can be transient and unreachable on the network, we probably do not want the entire matching operation to fail, like it would here if you were to return he error upstream. 
Instead lets log the error and return an empty result set.